src/back/wgsl/writer.rs

```diff
@@ -1563,6 +1567,16 @@ impl<W: Write> Writer<W> {
                         self.write_expr(module, arg, func_ctx)?;
                         write!(self.out, "))")?;
                     }
+                    Function::FindLsb => {
+                        write!(self.out, "((ctz(")?;
+                        self.write_expr(module, arg, func_ctx)?;
+                        write!(self.out, ") + 1) % 33) - 1")?;
+                    }
+                    Function::FindMsb => {
+                        write!(self.out, "((clz(")?;
+                        self.write_expr(module, arg, func_ctx)?;
+                        write!(self.out, ") + 1) % 33) - 1")?;
```

Comment:
something isn't right here. The implementations for `FindLsb` and `FindMsb` look the same?

Reply:
Note the "t" vs. "l" in `ctz` vs. `clz`

Reply:
Relatedly, since these functions aren't officially added to WGSL yet someone should seriously consider other names for them that are easier to tell apart!

Reply:
Please comment about this on the upstream issue as well!